src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -11,38 +12,73 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 import java.io.IOException;
+import java.time.Instant;
+import java.util.UUID;
 
 public class ProfileServletTest {
```

Comment:
Lets add tests to check the sanitation of the string.

Reply:
Done!